<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendWindow</class>
 <widget class="QMainWindow" name="FriendWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Friends</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">background-color : rgb(90, 98, 255)</string>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="AddFriendButton">
         <property name="text">
          <string>Add Contact</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(85, 255, 255)</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(85, 255, 255)</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(85, 255, 255)</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(85, 255, 255)</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(85, 255, 255)</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(255, 56, 21)</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(255, 56, 21)</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
